app/test_main.py

```diff
@@ -1 +1,61 @@
-# write your code here
+import unittest
+from unittest.mock import patch, Mock
```

Comment:
It will be better to do smth like this
`from unittest import mock, TestCase`

And then use `mock.patch`, because a lot of modules can have such name of function